jdbc/src/main/java/org/apache/iotdb/jdbc/IoTDBStatement.java

```diff
@@ -287,6 +288,10 @@ private boolean executeSQL(String sql) throws TException, SQLException {
         resultSet = databaseMetaData.getNodes(Constant.COUNT_NODES, path, null, null, level);
         return true;
       }
+    } else if(sqlToLowerCase.startsWith(SHOW_VERSION_COMMAND_LOWERCASE)) {
```

Comment:
startWith? So if I query "show version xxxxxx", it can also run successfully. Is that right?

Reply:
done